.github/workflows/test.yml

```
@@ -8,6 +8,9 @@ on:
 
   workflow_dispatch:
 
+env:
+  INVERT_PATH_VARIABLE: true

```

Some comment would be nice. Maybe a more meaningful name would?
```suggestion
  # Workaround to avoid usage of system wide installations
  USER_PATH_FIRST: true

```